java/ql/src/Security/CWE/CWE-113/ResponseSplitting.qhelp

```diff
@@ -5,16 +5,24 @@
 
```

Comment:
Should this file have it's name change too?

Reply:
Same as above.